exec/exec.go

```diff
@@ -79,3 +82,73 @@ func Run(ctx context.Context, command string, arg []string) (*CommandExecutor, e
 
 	return commandExecutor, nil
 }
+
+// Output returns stdout, stderr and error status for a given shell command
+func Output(ctx context.Context, command []string, timeout ...string) ([]byte, []byte, error) {
+
+	var (
+		// define an empty cancel function
+		cancel context.CancelFunc = func() {}
+		arg    []string
+	)
+
+	if ctx == nil {
+		// default to 10 seconds timeout.
+		ctx, cancel = context.WithTimeout(context.Background(), time.Second*10)
+	}
+
+	if len(command) == 0 {
+		return nil, nil, errors.New("unable to execute a command with empty Cmd field")
+	}
+
+	if timeout != nil {
```

Comment:
this is not the right way to compare strings in go
you should do
`if timeout != "" {`
or
`if len(timeout) == 0 {`